wdl2cwl/tests/wdl_files/gatk_1.wdl

```diff
@@ -0,0 +1,106 @@
+version 1.0
+
+# Source: https://github.com/biowdl/tasks/blob/bc1bacf11498d2d30b85591cfccdcf71ef0966a5/gatk.wdl#L980
+#
+# Copyright (c) 2018 Leiden University Medical Center
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+task HaplotypeCaller {
+    input {
+        Array[File]+ inputBams
+        Array[File]+ inputBamsIndex
+        String outputPath
+        File referenceFasta
+        File referenceFastaIndex
+        File referenceFastaDict
+        Boolean gvcf = false
+        #String emitRefConfidence = if gvcf then "GVCF" else "NONE"
+        Boolean dontUseSoftClippedBases = false
+
+        #Array[File]+? intervalList
+        #Array[File]+? excludeIntervalList
+        #Float? contamination
+        #File? dbsnpVCF
+        #File? dbsnpVCFIndex
+        #File? pedigree
+        #Int? ploidy
+        #String? outputMode
+        #Float? standardMinConfidenceThresholdForCalling
+
+        Int javaXmxMb = 4096
+        # Memory increases with time used. 4G should cover most use cases.
+        #Int memoryMb = javaXmxMb + 512
+        Int timeMinutes = 400 # This will likely be used with intervals, as such size based estimation can't be used.
+        String dockerImage = "quay.io/biocontainers/gatk4:4.1.8.0--py38h37ae868_0"
+    }
+
+    command {
+        set -e
+        mkdir -p "$(dirname ~{outputPath})"
+        gatk --java-options '-Xmx~{javaXmxMb}M -XX:ParallelGCThreads=1' \
+        HaplotypeCaller \
+        -R wd2/~{basename(referenceFasta)} \
```

Comment:
Good that you copied in the `wd2/` change, but this doesn't work without the `ln -s` changes that go with it https://github.com/biowdl/tasks/pull/291/files#diff-c76c01f3ca967cdb9c157a75e7fb1a08d0037543b455c2107398601a2f526ebfR1084

Reply:
Yes. I only saw those changes later :/